stackdriver.go

```diff
@@ -56,6 +56,7 @@ import (
 	"time"
 
 	traceapi "cloud.google.com/go/trace/apiv2"
+	monitoredres "contrib.go.opencensus.io/exporter/stackdriver/monitoredresources"
```

Comment:
Why not just import without aliasing the name?

Reply:
removed the alias